share/streams/shr_dmodel_mod.F90

```diff
@@ -383,7 +383,7 @@ subroutine shr_dmodel_readgrid( gGrid, gsMap, nxgo, nygo, nzgo, filename, compid
          nxgo = scm_nx
          nygo = scm_ny
          nzgo = -1
-         gsize = abs(scm_nx*scm_nx*nzgo)
+         gsize = abs(scm_nx*scm_ny*nzgo)
```

Comment:
I'm happy to approve, just a sanity question; do we need to make a similar change in the C++ code?  I'm guessing no.

Reply:
This change is applicable to both v0 and v1